<launch>

    <rosparam command="load" file="$(find fmt_star)/config/config.yaml" />
    <node pkg="fmt_star" name="fmt_star_server" type="fmt_star_server" output="screen"/>

</launch>